proposals/lip-0022.md

```diff
@@ -177,25 +177,17 @@ Internally, the algorithm works as follows:
 
 In the case that `ListStandbyDelegates` contains less than 2 elements, the delegates with rank 102 and 103 will be chosen as `delegate1 `and `delegate2` respectively.
 
-### Round Computation and Delegate List Ordering
+### Round Computation and Forger List
 
-The round computation is performed in the same way as in the current implementation with the difference that `delegate1` and `delegate2`, as defined in the previous section, are included in the forging delegate list of `round` + 1. This implies the following:
+A round is composed of 103 blocks now.
 
-*   The round length is now 103 block slots. Currently, this is defined by the constant  <code>[ACTIVE_DELEGATES](https://github.com/LiskHQ/lisk-sdk/blob/310283408102ff9f891830225afe8124e833a608/config/default/constants.js#L55)</code>.  For clarity, this constant should be replaced for three constants defining the number of the top 101 delegates by `delegateWeight`, the number of standby delegates, i.e., 2, and the length of the round (sum of the two previous constants).
-*   The change in the length of the round affects the logic to check when a round is finished which is 103 block slots in total.
-*   `delegate1` and `delegate2` have to be included in the delegate list<sup>3</sup> before it is re-ordered for `round` + 1. Currently, the function <code>[getKeysSortByVote](https://github.com/LiskHQ/lisk-sdk/blob/7c31c9d4b12a72f18a35ead79198b2ba6d4bd091/framework/src/modules/chain/submodules/delegates.js#L32)</code> computes this list before being re-ordered.
+Before being [shuffled](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0003.md), the forger list<sup>3</sup> for `round` + 1 is constructed as follows:
 
-Bear in mind that this proposal does not change any other point in the round processing once the list with the 103 forging delegates for the next round is computed.  
+* The addresses of the top 101 delegates at the end of `round` - 1 by lexicographic order according to the combined key `(delegateWeight, address)` are included in the forging list.
```

Comment:
```suggestion
* The addresses of the top 101 delegates by delegate weight at the end of `round - 2` are added to the forger set. Ties are broken in favor of smaller address. 
```

Reply:
- I think it should be the end of `round-2`. The same weights that are used for the standby delegate selection above.
- Sorting lexicographically by combined key `(delegateWeight, address)` is a bit confusing, because the delegate with smallest weight come first and it is unclear what top means then. Did you want to do the tie-breaking by larger address?

Reply:
True, the round was wrong. Also modified the bullet point to make it less confusing.